components/style/values/computed/image.rs

```diff
@@ -54,6 +54,8 @@ pub enum LineDirection {
     Angle(Angle),
     /// A corner.
     Corner(X, Y),
+    /// A Position and an Angle for legacy `-moz-` prefixed gradient.
+    PositionAngle(Option<Position>, Option<Angle>),
```

Comment:
Currently `-webkit-` prefixed gradients are also a servo gradient type. And I assumed we would want these in servo too. Am I wrong?